README.md

```diff
@@ -138,6 +138,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [NepXpress Cargo & Courier](https://www.nepxpress.com) | Buddhanagar, Kathmandu | Instant Home Delivery Service in Nepal
 - [NITV Streamz Pvt. Ltd.](http://nettv.com.np/) | Ekantakuna, Lalitpur | TV streaming
 - [nLocate](https://nlocate.com) | Harihar Bhawan, Lalitpur | Locate services, businesses, events around you
+- [Numeric Mind](https://numericmind.com) | Now near Bhatbhateni, Kathmandu | A Statistical Service Company in Nepal
```

Comment:
@bjungbogati Thank you for the PR.

Can you change the location to be `Bhatbhateni, Kathmandu`? And ds there a better description for the company?

Reply:
@mesaugat I have updated both locations and descriptions. Thanks